qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -684,3 +742,41 @@ QImage HOCRPdfExporter::getSelection(const QRect& bbox)
 {
 	return m_displayerTool->getSelection(bbox);
 }
+
+void HOCRPdfExporter::paperSizeChanged() {
+	PaperSizeMetrics currentMetric = static_cast<PaperSizeMetrics>(ui.comboBoxPaperSizeMetric->itemData
+			(ui.comboBoxPaperSizeMetric->currentIndex()).toInt());
+
+	auto textPaperSize = ui.comboBoxPaperSize->itemData(ui.comboBoxPaperSize->currentIndex()).toString();
+	if(textPaperSize == "Custom") {
+		ui.lineEditPaperSizeWidth->setEnabled(true);
+		ui.lineEditPaperSizeHeight->setEnabled(true);
+	} else if(textPaperSize == "Same as source") {
```

Comment:
I'd not use human labels for logic (it is fragile, because you can mistake it for a UI label and changing it causes all sorts of troubles). Just use "source" and "custom".

Reply:
Fixed.